client/src/app/(modules)/network/(details)/field.tsx

```diff
@@ -72,7 +72,7 @@ const Field = ({ label, value, url, type, hasEllipsis }: Field & { type: Type })
             {value}
           </div>
           {isOverTwoLines && (
-            <button onClick={toggleExpanded} className="text-sm font-semibold text-mod-sc-ev">
+            <button onClick={toggleExpanded} className="text-sm font-semibold text-green-700">
```

Comment:
I think that Lidia's intention is to have different colours depending on what is displayed:

- Organisation: #358FE3
- Project: #EF6A4C

Reply:
On Network, I think the “Suggest changes” and “Visit website” buttons should also always be gray. Thanks!